src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -1155,8 +1157,12 @@ namespace DataHeatBalance {
         int PlenumCondNum;                        // Supply or return plenum conditions number, 0 if this is not a plenum zone
         int TempControlledZoneIndex;              // this is the index number for TempControlledZone structure for lookup
         //            Pointers to Surface Data Structure
-        int SurfaceFirst;          // First Surface in Zone
-        int SurfaceLast;           // Last Surface in Zone
+        int SurfaceFirst;          // First Heat Transfer Surface in Zone
+        int SurfaceLast;           // Last  Heat Transfer Surface in Zone
+        int NonWindowSurfaceFirst; // First Non-Window Heat Transfer Surface in Zone
+        int NonWindowSurfaceLast;  // Last  Non-Window Heat Transfer Surface in Zone
+        int WindowSurfaceFirst;    // First Window Heat Transfer Surface in Zone
+        int WindowSurfaceLast;     // Last  Window Heat Transfer Surface in Zone
```

Comment:
These are in the `Zone` struct and govern the loops in `CalcHeatBalanceInsideSurf2CTFOnly `.